<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientForm</class>
 <widget class="QWidget" name="ClientForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(162, 162, 162);
font: 75 13pt &quot;Arial&quot;;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_client">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>7</number>
      </property>
      <property name="topMargin">
       <number>7</number>
      </property>
      <property name="rightMargin">
       <number>7</number>
      </property>
      <property name="bottomMargin">
       <number>7</number>
      </property>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_info">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>36</height>
           </size>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnClear">
          <property name="minimumSize">
           <size>
            <width>45</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>99</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>清除数据</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>38</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(85, 255, 0);
font: 75 16pt &quot;Arial&quot;;</string>
            </property>
            <property name="text">
             <string>OK</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_ok">
            <property name="styleSheet">
             <string notr="true">color: rgb(85, 255, 0);
font: 75 16pt &quot;Arial&quot;;</string>
            </property>
            <property name="text">
             <string>0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>38</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 0, 0);
font: 75 16pt &quot;Arial&quot;;</string>
            </property>
            <property name="text">
             <string>NG</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_ng">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 0, 0);
font: 75 16pt &quot;Arial&quot;;</string>
            </property>
            <property name="text">
             <string>0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="minimumSize">
             <size>
              <width>51</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 85, 255);
font: 75 16pt &quot;Arial&quot;;</string>
            </property>
            <property name="text">
             <string>误判</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_error">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 85, 255);
font: 75 14pt &quot;Arial&quot;;</string>
            </property>
            <property name="text">
             <string>0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,4">
          <property name="spacing">
           <number>7</number>
          </property>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="minimumSize">
             <size>
              <width>51</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 85, 255);
font: 75 16pt &quot;Arial&quot;;</string>
            </property>
            <property name="text">
             <string>漏判</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_miss">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 85, 255);
font: 75 16pt &quot;Arial&quot;;</string>
            </property>
            <property name="text">
             <string>0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_status">
          <property name="minimumSize">
           <size>
            <width>99</width>
            <height>39</height>
           </size>
          </property>
          <property name="text">
           <string>离线</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSwitch">
          <property name="minimumSize">
           <size>
            <width>45</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>99</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>关</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
